analysis_options.yaml

```diff
@@ -0,0 +1,19 @@
+include: package:flutter_lints/flutter.yaml
```

Comment:
you are including `flutter_lints` here, but it's not imported in `pubspec.yaml`. Instead, you imported `lints` at https://github.com/fluttercommunity/import_sorter/pull/75/files#diff-8b7e9df87668ffa6a04b32e1769a33434999e54ae081c52e5d943c541d4c0d25R21.

This line should be updated to
```yaml
include: package:lints/recommended.yaml
```